CHANGELOG.md

```diff
@@ -42,6 +42,7 @@
 - **Breaking:** On Web, the canvas element associated to a `Window` is no longer removed from the DOM when the `Window` is dropped.
 - On Web, `WindowEvent::Resized` is now emitted when `Window::set_inner_size` is called.
 - **Breaking:** `Fullscreen` enum now uses `Borderless(Option<MonitorHandle>)` instead of `Borderless(MonitorHandle)` to allow picking the current monitor.
+- On MacOS, convert location to physical unit in `emit_move_event`.
```

Comment:
```suggestion
- On macOS, fix `WindowEvent::Moved` coordinates when scale factor isn't 1.
```

Reply:
I think it's worth it to describe the error, rather than just saying they're fixed when the scale factor isn't 1.

```suggestion
- On MacOS, fix `WindowEvent::Moved` ignoring the scale factor.
```